config/software/cacerts.rb

```diff
@@ -16,69 +16,26 @@
 
 name "cacerts"
 
-# Date of the file is in a comment at the start, or in the changelog
-default_version "2016.01.20"
+# We always pull the latest version,
+# so the hashsum check will break every time the file is updated on the remote
```

Comment:
If at some point I get too tired of updating the checksum I'll probably just pin the version...